.github/dependabot.yml

```
@@ -13,24 +13,52 @@ updates:
     # We don't trigger Jenkins or GitHub Actions builds on pull requests from dependabot,
     # so we can safely use a high limit here.
     open-pull-requests-limit: 20
+    groups:
+      # This group combines all build-only dependencies. Published artifacts do not depend on them.
+      # Grouping such dependencies will make Dependabot create PRs with a branch name
+      # following the pattern (`dependabot/maven/build-dependencies-.*`)
+      # and with a title like `Bump the build-dependencies group with 8 updates` that we can easily
+      # use for Hibernate Bot rules.
+      build-dependencies:
+        patterns:
+          # Maven plugin patterns:
+          - "*plugin*"
+          - "*maven*"
```

I think the first one is a bit dangerous, as it's not unthinkable that a dependency of ours would have plugin in its name. I could imagine a `jackson-something-plugin` dependency in the Elasticsearch Sniffer, for example.

Maybe this would be more reasonable?

```suggestion
          - "*maven*plugin*"
```

---

mmm... probably yes 😃. We had this rule before:
```
".*\\b(maven|surefire)\\b.*\\bplugin\\b.*"
```
but as we have a line for `surefire`, it is probably safe to combine it as you are suggesting. And then we'll wait and see what Dependabot will come up with that doesn't fit the patterns 😈😃